lib/terminus.spec.js

```diff
@@ -254,6 +254,45 @@ describe('Terminus', () => {
       })
     })
 
+    it('includes error on reject (Error objects)', async () => {
+      let onHealthCheckRan = false
+
+      const errors = [
+        new Error('test error 1'),
+        new Error('test error 2')
+      ]
+
+      createTerminus(server, {
+        healthChecks: {
+          '/health': () => {
+            onHealthCheckRan = true
+            const myError = new HealthCheckError('failed', errors)
+            return Promise.reject(myError)
+          }
+        }
+      })
+      server.listen(8000)
+
+      const res = await fetch('http://localhost:8000/health')
+      expect(res.status).to.eql(503)
+      expect(onHealthCheckRan).to.eql(true)
+      const json = await res.json()
+
+      const errorMessages = [{
+        message: errors[0].message,
+        stack: errors[0].stack.toString()
```

Comment:
this can be a potential [security problem](https://owasp.org/www-community/Improper_Error_Handling)

I am ok with returning the name of the errors, but not the stack traces

Reply:
Thanks @gergelyke - what about if we paid attention to the `verbatim` variable and by default exclude, and include if `verbatim: true` ?

Reply:
hm, that may work - however, I am worried that it would be a change of behavior, and some people won't expect that we are returning stack traces

The way I see it, we have two options:

- Release a breaking version to do this (I really don't want to do that)
- Add an additional `__unsafeExposeStackTraces` configuration option (or something similar), if you have a strong use-case for having stack traces exposed

is there anything else I am missing?